notebooks/1-sequence-stats.jmd

```diff
@@ -0,0 +1,147 @@
+using Plots: make_fillrange_from_ribbon
+# Coronavirus Analysis
+Working directory must be Cov2_Analysis directory
+
+## Mean and Standard Deviation
+Finding mean sequence length, standard devation for length, mean gc content and standard deviation for gc content for CoV sequences.
+
+```julia 
+using BISC195Bioinformatics
+
+return mean_and_std_fasta("data/CoV_sequences.fasta")
```

Comment:
`return` only works from inside function calls. Pretend these code blocks are a REPL - you can just call the function